packages/web-app/src/middleware/check-appeal-outcome.spec.js

```diff
@@ -18,10 +18,7 @@ describe('middleware/check-appeal-outcome', () => {
       title: 'Return next if there is appeal id in the params',
```

Comment:
It doesn't appear that you're using params anymore...

Reply:
Addressed via commit [0a79768](https://github.com/Planning-Inspectorate/back-office/pull/83/commits/0a797681b995faa4d2229ae2b0a99b192eaa6f5d)